echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/orca/PipelineInitiator.java

```diff
@@ -93,7 +96,13 @@ public void initialize() {
     }
   }
 
-  public void startPipeline(Pipeline pipeline) {
+  public enum TriggerSource {
+    SCHEDULER,
+    MISSEDSCHEDULER,
+    EVENT
```

Comment:
might be interesting to have a different value for the different types of events

Reply:
as in jenkins vs. docker, etc?

Reply:
yep

Reply:
added to `logOrcaErrorMetric`